src/truefoundry_sdk/types/agent_manifest.py

```
@@ -2,51 +2,8 @@
-    owned_by: typing_extensions.Annotated[typing.Optional[OwnedBy], FieldMetadata(alias="ownedBy")] = pydantic.Field(
-        alias="ownedBy", default=None
-    )
-
-    if IS_PYDANTIC_V2:
-        model_config: typing.ClassVar[pydantic.ConfigDict] = pydantic.ConfigDict(extra="allow")  # type: ignore # Pydantic v2
-    else:
-
-        class Config:
-            smart_union = True
-            extra = pydantic.Extra.allow
+AgentManifest = typing.Union[TrueFoundryAgentManifest, RemoteAgent, LegacyAgentManifest]
```

Ruff (0.15.9)

[warning] Use `X | Y` for type annotations

(UP007)